index.d.ts

```diff
@@ -0,0 +1,77 @@
+export as namespace SlowJsonStringify;
+
+export type JSONPrimitive =
+  string
+  | number
+  | boolean
+  | null;
+
+export type AttrType =
+  'string'
+  | 'number'
+  | 'boolean'
+  | 'array'
+  | 'null';
+
+/**
+ * SjsSchema
+ * 
+ * Schema that describe the structure of your data.
+ * 
+ * Usage: https://github.com/lucagez/slow-json-stringify#usage
+ */
+export type SjsSchema = { [prop: string]: AttrExecutable | SjsSchema };
+
+/**
+ * SjsSerializer
+ * 
+ * Serialize any object enforcing the provided schema.
+ */
+export type SjsSerializer = (obj: object) => string;
```

Comment:
It would be better to use unknown here instead of object.
`export type SjsSerializer = (obj: unknown) => string;`